### DISCUSSION TITLE
Gene Fusion Definition

### DISCUSSION THREAD

**Discussion by ahwagner** [HGVSnomenclature/hgvs-nomenclature#10] [HGVS nomenclature proposals]
The definition of a gene fusion is based on the work of the cross-consortia Gene Fusion project hosted by the VICC at [fusions.cancervariants.org](https://fusions.cancervariants.org/). Alignment on this definition is critical for describing gene fusions and associated recommendations for gene fusion nomenclature.

_A gene fusion is defined as the joining of two or more genes to drive aberrant activity of a gene product, through formation of a chimeric transcript or interaction of rearranged gene regulatory elements._ ([source](https://fusions.cancervariants.org/en/latest/terminology.html#gene-fusions))

This definition was selected to capture the most common molecular entities referred to as fusions, and to clearly differentiate the fusion (the gene product) from the mechanism that drives it (rearrangement, trans-splicing, etc.). The definition is inclusive of archetypal chimeric transcripts such as `BCR::ABL1`, promoter-swap fusions such as `reg_p@TMPRSS2::ERG`, and enhancer-driven fusions such as `reg_e@GATA2::EVI1`. It also allows for the description of chimeric transcript products derived from transcriptional processing events, including known trans-splicing (e.g. `JAZF1::JJAZ1`) and read-through (e.g. `CTSD::IFITM10`) fusions.

These two classes of fusions are defined formally as:

**Chimeric transcript fusion**: _A chimeric transcript is a transcript composed of exons from two or more genes._ ([source](https://fusions.cancervariants.org/en/latest/terminology.html#chimeric-transcript-fusions))

**Regulatory fusion**: _A regulatory fusion is the interaction of regulatory elements between two genes brought into proximity by genomic rearrangement, driving aberrant gene product expression._ ([source](https://fusions.cancervariants.org/en/latest/terminology.html#regulatory-fusions))

We would like these definitions formally adopted in the HGVS and ISCN nomenclatures and the HGNC guidelines.

**Current status:** 
Addressed in https://github.com/HGVSnomenclature/VarNomen/pull/34

**Comment by jtdendunnen** [comment]
Assume the case of the insertion of a copy of a genomic segment (containing the 3' half of gene B) into an intron of gene A (same transcriptional orientation. As a consequence, transcription of gene A now terminates at the end of the inserted gene B. Is this event considered as a "gene fusion"?

**Reply by ahwagner** [reply]
Yes, this would result in a chimeric transcript product and be considered a gene fusion

**Comment by syeol13** [comment]
我已经收到您的邮件，将尽快给您回复。祝好！

**Comment by elspethb** [comment]
> We would like these definitions formally adopted in the HGVS and ISCN nomenclatures and the HGNC guidelines.

Just to clarify, HGNC have no formal activity "naming" gene fusions, so we wouldn't incorporate any definitions into our gene naming guidelines.

**Comment by RosHastir** [comment]
I have had feedback from several different parts of the genomics community, including ISCN committee, HGNC, diagnostic laboratories, global EQA providers and specialists in haematology.

1. The proposal that the notation for a fusion gene should include the identifier i.e. BCR(hgnc:1014)::ABL1(hgnc:76) is not consistent with ISCN, HGVS or the paper published by Bruford et al.. 2020.
2. The  addition of identifier e.g. hgnc:1014 make the notation/nomenclature cumbersome and feedback from the ISCN committee and diagnostic laboratories is that this would not be universally adopted. 
3. The genetics community is aware of the genes involved in the fusions and therefore there is no need for a duplication of information and/or the inclusion of a HGNC identifier in the notation as it doesn't add any additional information nor does it increase clarity.  
4. This change would require a major upheaval of internal reporting systems and databases. 
5. The inclusion of the identifier is not an essential requirement of the notation. This identifier information can be given in the text of the publication or diagnostic report if needed.  This is consistent with other identifiers used in genomic testing.  The current ISCN & HGVS nomenclature includes the genome build so the gene names are correct within this context.
6. Finally, the use of a colon in the identifier infers a break in ISCN and therefore the ISCN committee strongly advised against the inclusion of a colon in the identifier.


Ros Hastings
Chair of the ISCN Committee

**Comment by chromosomix** [comment]
On behalf of GFCH (Groupe Francophone de Cytogenetique Hématologique), I report their suggestion on that topic.


We agree with the WHO-HAEM5 co-authors’ suggestion of using the new nomenclature proposed by Bruford et al. [5], in which a double colon (::) describes all gene rearrangements – both those leading to a chimeric gene and those in which the regulatory or enhancer components of one gene deregulate a partner gene.
Even though the term “fusion” was suggested by Bruford et al. and has been adopted in WHO-HAEM5, we consider that “juxtaposition” is more suitable when referring to the deregulation of an intact oncogene. Alaggio et al. used the term “fusion” for the IGH::CCND1 rearrangement in mantle cell lymphoma, and “juxtaposition” for the IG::MYC rearrangement in Burkitt lymphoma[1]. These examples highlight possible confusion in understanding oncogenic mechanisms. 
We therefore suggest that the words “fusion” and “juxtaposition” should be used, given the underlying molecular mechanisms of the corresponding translocations.

Summary : 
all gene rearrangements leading to a chimeric gene : fusion
all gene rearrangements  in which the regulatory or enhancer components of one gene deregulate a partner gene : juxtaposition

Jean-Michel Dupont, 
Member of ISCN Committee

**Reply by ahwagner** [reply]
Thanks @chromosomix. It is good to see the GFCH calling for this disambiguation, which the VICC effort has strived to do both in the above definitions and our [regulatory element extensions](https://fusions.cancervariants.org/en/latest/nomenclature.html#regulatory-nomenclature) to the HGNC recommendations for fusion descriptions. 

I will read up on the findings of the GFCH to inform our alignment effort; a Google Scholar search revealed [this publication](https://www.nature.com/articles/s41375-023-01821-3) as the source of the above recommendations, please let me know if there are additional references I should review.

Also, I brought up these comments to the group on our 2/8 call while reviewing other feedback from HGNC/ISCN/HVNC, and there was an interest in understanding how "juxtaposition"and "fusion" should be used in the case of "[promoter-swap](https://fusions.cancervariants.org/en/latest/terminology.html#regulatory-fusions)" variants, where the transcript promoter and part of the 5' UTR for Gene A is adjoined to the 5' UTR for Gene B.

**Comment by elspethb** [comment]
We've noted that you've stated "It also allows for the description of chimeric transcript products derived from transcriptional processing events, including known trans-splicing (e.g. JAZF1::JJAZ1) and read-through (e.g. CTSD::IFITM10) fusions." 

As stated in "HUGO Gene Nomenclature Committee (HGNC) recommendations for the designation of gene fusions":
The HGNC has approved the use of the hyphen separator, in collaboration with all contributing genome annotation groups involved in the Consensus Coding Sequence (CCDS) Project [[12](https://www.nature.com/articles/s41375-021-01436-6#ref-CR12)], for denoting readthrough transcripts, e.g., INS-IGF2.

Therefore, we do not endorse the usage of the double colon for denoting transcripts resulting from readthrough of adjacent genes - ie the "::" is to denote fusions generated by chromosomal rearrangements, as stated in the recommendations paper.

**Reply by ahwagner** [reply]
Thanks for highlighting this issue, which we had not discussed in-depth prior to you raising it. We have reviewed your concern and the cited text, and are working to accommodate this in our nomenclature.

**Comment by marinadistefano** [comment]
As I am not an expert in gene fusions, I defer to those members of HGNC and ISCN and their proposals above.

**Comment by jtdendunnen** [comment]
Is the addition of "to drive aberrant activity of a gene product, through formation of a chimeric transcript or interaction of rearranged gene regulatory elements" required. It might be difficult to detect a "chimeric transcript" esp. when it is instable. Is the gene fusion dependent on the detection of a chimeric transcript?  The definition of a "chimeric transcript" is missing as well.

**Reply by ahwagner** [reply]
It is the chimeric and/or deregulated gene product being described that defines a gene fusion. It is broadly agreed that a rearrangement that does not result in a stable gene product should not be described as a fusion. However, we also do not think that detection of a fusion is necessary to describe one if other contextual information supports the presence of a fusion, e.g. calling a BCR::ABL1 fusion with the detection of a compatible genomic rearrangement in a chronic myelogenous leukemia tumor.

**Reply by jtdendunnen** [reply]
So your definition is a RNA level definition, not for DNA level ?

**Reply by ahwagner** [reply]
It is components of both of these contexts, together, that define gene fusions. Our definition involves two or more genes defined at the DNA level, and an associated gene product at the RNA level.

**Comment by jtdendunnen** [comment]
I do not understand the "through [] interaction of rearranged gene regulatory elements". Should "of" be "with a"?  In globin genes there is the example of a single nucleotide variant creating a new regulatory element. Would this be a gene fusion?

**Reply by ahwagner** [reply]
Ah, yes, I think that last clause can be improved upon for clarity. We are revising the terminology for clarity in response to other feedback from ISCN / HGNC / HVNC and I will try to improve on the wording of this.

Rearrangements are part of the definition of a regulatory fusion, which is what distinguishes it from scenarios like the globin gene example provided here.

**Reply by jtdendunnen** [reply]
So there is a minimal length for a rearrangement?  In the globin example the single nucleotide variant shift transcription from gene B to gene A which lies upstream of gene B.

**Reply by ahwagner** [reply]
No, we are not defining rearrangements by length, nor are we focused on describing them except in contrast to gene fusions. The regulatory fusion definition:

> A regulatory fusion is defined as the interaction of regulatory elements between two genes brought into proximity by genomic rearrangement, driving aberrant gene product expression.

The distinction I would make for aligning on the fusion terminology is not "what length of molecular sequence change constitutes a rearrangement", but "what constitutes bringing genes into proximity by genomic rearrangement". In the provided example, a SNV does not move a gene regulatory element nearer another gene, and so I would say it is not a gene fusion by our definition.
